sdk/lib/opentelemetry/sdk/trace/tracer.rb

```diff
@@ -9,6 +9,8 @@ module SDK
     module Trace
       # {Tracer} is the SDK implementation of {OpenTelemetry::Trace::Tracer}.
       class Tracer < OpenTelemetry::Trace::Tracer
+        DEFAULT_SPAN_NAME = 'empty'
```

Comment:
There's no real value in extracting this named constant - it is only used in one place, and this makes it *less* readable.

Reply:
reverted this change